_targets.R

```diff
@@ -24,6 +24,35 @@ if (Sys.getenv("GCS_AUTH_FILE") != "") {
   repository <- targets::tar_option_get("repository")
 }
 
+# Model parameters
+dflt_pa <- c("log_pa_volume", "scale_pa_intensity")
+scale_pa <- c("scale_pa_volume", "scale_pa_intensity")
+dflt_sleep <- c(
+  "scale_sleep_duration", "scale_sleep_efficiency", "scale_sleep_onset",
+  "scale_sleep_regularity"
+)
+dflt_ranef <- "(1|studyid) + (1|participant_id)"
+fixedef <- "(1|measurement_day) + (1|participant_id)"
```

Comment:
Should we ditch the random intercept for measurement_day here as everyone starts on different days?

Reply:
Yeah, good point. I've removed it.

As an aside - it's cool that if you only change one row of the model_df it only invalidates the targets related to that row.